invenio_rdm_records/oaiserver/services/results.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Graz University of Technology.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Result items for OAI-PMH services."""
+
+from invenio_records_resources.pagination import Pagination
```

Comment:
Comment: Not something to fix now for this PR, but I think we need to move some of this code to Records-Resources, and see how we can make it more reusable. I've seen it slightly duplicated many  places with minor modifications. Again, this is more a comment for myself :-)